bentoctl_container_instances/templates/terraform_default.tf

```diff
@@ -0,0 +1,118 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.97.0"
+    }
+  }
+}
+
+# Configure the Microsoft Azure Provider
+provider "azurerm" {
+}
+
+################################################################################
+# Input variable definitions
+################################################################################
+
+
+variable "deployment_name" {
+  description = "Name of the deployment."
+  type        = string
+}
+
+variable "image_tag" {
+  description = "full image gcr image tag"
+  type        = string
+}
+
+variable "image_repository" {
+  description = "gcr repository name"
+  type        = string
+}
+
+variable "image_version" {
+  description = "gcr image version"
```

Comment:
Let's fix this description in the follow up documentation PR. Same for Heroku deployment